cpp_test_suite/cpp_test_ds/DevTest.cpp

```diff
@@ -217,6 +217,11 @@ void DevTest::always_executed_hook()
 
 }
 
+void DevTest::server_init_hook()
+{
+    cout2 << "In server_init_hook method" << endl;
```

Comment:
Consider using `std::endl`, otherwise it may break with #528 (general comment, applies to other places as well)

Reply:
This is a DevTest - a part of cpp_test_suite, so I think it is not relevant here to use `std::` as anyway in other parts of this class it is missing. I would wait till #528 is merged and then fix the issues if any.